warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reduce/ColumnTransform.java

```diff
@@ -0,0 +1,92 @@
+package datawave.ingest.mapreduce.job.reduce;
+
+import java.nio.charset.StandardCharsets;
```

Comment:
See https://github.com/NationalSecurityAgency/datawave/pull/2534

Reply:
Actually, it appears you have everything required here, so I am going to close the other one.